conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+
+            <ul>
+                <li>Get help and support to safely resolve issues that negatively affect them</li>
+                <li>Correct their behaviours that negatively affect other participants in W3C</li>
+                <li>Continue to safely work and contribute to the W3C after an issue has been raised</li>
+            </ul>
+                
+            <p>The Code of Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+                
+            <p>All situations are different and W3C participants are encouraged to use whatever method of conflict resolution they feel comfortable with.</p>
+                
+            <p>Participants are encouraged to try and resolve issues themselves in the first instance. However, if participants do not feel comfortable or safe doing so, they can get help through an ombudsperson, facilitators and/or mediation.</p>
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Participants should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the conflict resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
+            </ul>
+                
+            <p>A disciplinary investigation can be started where one of the following apply:</p>
+                
+            <ul>
+                <li>It’s not appropriate to begin with mediation</li> 
+                <li>All other mediation or resolution attempts have not worked</li>
+            </ul>
+        </section>
+
+        <section id="procedures">
+            <h2>Conflict Resolution Procedures</h2>
+
+            <p>When an incident arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the incident, feel supported and respected.</p>
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
```

Comment:
Before the "Incident is reported" item, I think it would be good to insert a step for "Incident is discussed directly with the parties involved".  I think it would also be helpful to provide a little guidance about how to do this most effectively.  For example, in the case of a person who inadvertently make someone else feel bad or slighted, I think it is helpful to both: 1. explain how it made the recipient feel and why; and 2. request a specific different approach in the future.   I think[ the example given in CoC section 5](https://w3c.github.io/PWETF/#mistake) is pretty good, though it leaves out (as implied) the component of explaining how the incident made the recipient feel and why:

> Alice: “Yeah I used X and it was really crazy!” Eve: “Hey, could you not use that word? What about ‘ridiculous’ instead?” Alice: “oh sorry, sure.” -> edits old message to say “Yeah I used X and it was really confusing!”

Reply:
+1 I agree. It's essential to follow the best practices that allow individuals to resolve issues themselves before seeking external help. As per the project management best practice (PMBOK), individuals should be given a chance to resolve conflicts on their own. In our case, the Ombudsperson should remain impartial and use the PWE resources and Conflict Management Procedure to help the individuals initiate the conflict resolution process.